operator_ui/src/pages/Jobs/New.test.tsx

```diff
@@ -47,119 +40,72 @@ describe('pages/Jobs/New', () => {
   it('loads TOML spec definition from search param', () => {
     const expected = '"foo"="bar"'
 
-    const wrapper = mountWithProviders(
-      <Route path="/jobs/new" component={New} />,
-      {
-        initialEntries: [`/jobs/new?definition=${expected}`],
-      },
-    )
-
-    expect(wrapper.text()).toContain('"foo"="bar"')
-    expect(wrapper.text()).toContain(`TOML blob`)
-  })
+    renderWithRouter(<Route path="/jobs/new" component={New} />, {
+      initialEntries: [`/jobs/new?definition=${expected}`],
+    })
 
-  it('saves the spec in a storage', () => {
-    const wrapper = mountWithProviders(
-      <Route path="/jobs/new" component={New} />,
-      {
-        initialEntries: [`/jobs/new`],
-      },
+    expect(getByPlaceholderText('Paste TOML').textContent).toEqual(
+      '"foo"="bar"',
     )
-
-    expect(wrapper.text()).toContain(`TOML blob`)
   })
 
   describe('validate', () => {
     it('TOML format', () => {
-      expect(
-        validate({
-          value: '"foo"="bar"',
-        }),
-      ).toEqual(true)
-
-      expect(
-        validate({
-          value: '"foo""bar"',
-        }),
-      ).toEqual(false)
+      expect(validate({ value: '"foo"="bar"' })).toEqual(true)
+      expect(validate({ value: '"foo""bar"' })).toEqual(false)
     })
   })
 
-  it('updates toml preview task list', () => {
+  it('updates toml preview task list', async () => {
     const jobSpec1 =
       'observationSource = """ ds [type=ds]; ds_parse [type=ds_parse];  """'
-    const jobSpec2 =
-      'observationSource = """ ds [type=ds]; ds_parse [type=ds_parse]; ds_multiply [type=ds_multiply]; """'
-
-    const wrapper = mountWithProviders(
-      <Route path="/jobs/new" component={New} />,
-      {
-        initialEntries: [`/jobs/new`],
-      },
-    )
-    jest.useFakeTimers()
-    fillTextarea(wrapper, jobSpec1)
-    act(() => {
-      jest.runAllTimers()
-    })
-    wrapper.update()
+    // const jobSpec2 =
+    //   'observationSource = """ ds [type=ds]; ds_parse [type=ds_parse]; ds_multiply [type=ds_multiply]; """'
```

Comment:
Shouldn't have commented out code

Reply:
This has been uncommented now because it is used in the fixed test